test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -5169,6 +5169,9 @@ TEST_F(AtenXlaTensorTest, TestOneIndexTransfer) {
 }
 
 TEST_F(AtenXlaTensorTest, TestNonzero) {
+  // skip this test until the related bug(b/266636840) is fixed.
+  return;
+
```

Comment:
`test/cpp/test_aten_xla_tensor.cpp`'s `AtenXlaTensorTest, TestNonzero` will be temp disabled in https://github.com/pytorch/xla/pull/4510/files,

after #4510 merged, this PR will rebase

Reply:
merged in https://github.com/pytorch/xla/pull/4512